Makefile

```diff
@@ -15,8 +15,13 @@ deep-clean:
 
 # Run the application
 run:
```

Comment:
Why are we copying and overwriting our env file here?

Reply:
Hey. The .env.example file is provided in the wp-template repo. Once a site is created this file can be modified to reflect the SERVER_NAME of our local docker instance - it's important the the .env file isn't uploaded to our repo. The salts vars within .env do not need to be modified in order for the site to work locally however these are populated via wp-cloudformation-params when the site move through our stack.

The cp command won't have a negative impact on our local dev. In fact we get the additional efficiency of not having to worry about copying the example file each time we nuke and reload a repo.